heuristic_baseline.py

```diff
@@ -0,0 +1,44 @@
+from dataloader import *
+from model import heuristic
+import numpy as np
+import torch
+from matplotlib import pyplot as plt
+
+
+class heuristic_run:
+    """
+    The class to run the heuristic model. No training needed.
+    Parameters can be tuned in model.heuristic
+    """
+    def __init__(self):
+        self.dataloader = single_dataloader()
+        self.num_data = len(self.dataloader)
+        self.model = heuristic()
+        
+    def get_inputs_and_results(self, features, labels):
```

Comment:
If this were a more generalized method, features/labels should be stored inside `self.model`.

Reply:
This function is more or less a function to preprocess data for heuristic model, and features/labels are from dataloader, which I feel is usually not stored inside the model. However, I think this can be written as another collate_fn for dataloader, just like the PadSequence for LSTM. But I'm not exactly sure and need to test it first.

Reply:
I agree with you. On closer look, I think I misinterpreted what features and labels meant. In my head, I thought they were model features, not input features to make predictions on. 

In that case, this should be fine as is. Let's plan to talk more about what code needs to be generalized at our check-in this week.

Reply:
Yeah, that was a bad name for the function now I think about it. 

I'm also planning to discuss with you in detail on the generalization of the code. Right now, some generalized functions for each model in my head: 1. train(): train the model on the training set. Which should do nothing for the heuristic model, since it doesn't need to train. 2. get_accuracy():
get the accuracy on the test set, probably at different time instead of just at the end (which is what it is right now) 3. get_single_graph(): give a parsed game file as input, output the predicted probability at different times. This will be used as the deliverable, as the user may want to check the win prob graph for a specific game.

Sorry for the lengthy comment, but this is what's in my mind so far. We can discuss if these make sense or not during the check-in.

Reply:
Definitely, let's talk more during the check-in. Seems like you're on the right track :thumbsup:, so let's break down the design a little more live.